test/i18n_viz_test.rb

```diff
@@ -51,3 +51,36 @@ class I18nVizJavascriptIntegrationTest < ActionDispatch::IntegrationTest
     assert page.has_css?("#i18n_viz_tooltip")
   end
 end
+
+class I18nVizDynamicUrlTest < ActionDispatch::IntegrationTest
+  setup do
+    Capybara.current_driver = Capybara.javascript_driver # :selenium by default
+
+    # monkey patch external_tool_url
```

Comment:
hmm.  I would think there must be a better way.  It should be possible to pass in the lambda as a config option just like you do in a regular app.